CHANGELOG.md

```diff
@@ -2,16 +2,20 @@
 
 ## Grafana Mimir - main / unreleased
 
+## 2.5.0
+
 ### Grafana Mimir
 
 * [CHANGE] Flag `-azure.msi-resource` is now ignored, and will be removed in Mimir 2.7. This setting is now made automatically by Azure. #2682
 * [CHANGE] Experimental flag `-blocks-storage.tsdb.out-of-order-capacity-min` has been removed. #3261
 * [CHANGE] Distributor: Wrap errors from pushing to ingesters with useful context, for example clarifying timeouts. #3307
 * [CHANGE] The default value of `-server.http-write-timeout` has changed from 30s to 2m. #3346
 * [CHANGE] Reduce period of health checks in connection pools for querier->store-gateway, ruler->ruler, and alertmanager->alertmanager clients to 10s. This reduces the time to fail a gRPC call when the remote stops responding. #3168
+* [CHANGE] Hide TSDB block ranges period config from doc and mark it experimental. #3518
+* [CHANGE] Chore: use upstream rules.SendAlerts(). #3271
```

Comment:
What has changed from user's perspective? Why does Mimir user/admin care?

Reply:
Nothing has changed from user perspective. We can remove it.

Reply:
Fixed: 45b55e81d